.azure-pipelines/Linux-CI.yml

```diff
@@ -4,34 +4,34 @@ trigger:
 jobs:
 - job: 'Test'
   pool:
-    vmImage: 'Ubuntu-18.04'
+    vmImage: ubuntu-latest
```

Comment:
Could we still keep Ubuntu-18.04 here? To cover more environments, typically release CIs use the latest version of OS and AZP pipelines use an older version of OS. 

https://github.com/onnx/onnx/pull/4365 previously I made Linux-CI test with Ubuntu 18.04+protoc 3.0.0.

Reply:
ok!